### DISCUSSION TITLE
Compatibility with Obsidian's new Properties feature

### DISCUSSION THREAD

**Discussion by Acylation** [marcusolsson/obsidian-projects#636] [General]
Reference: https://obsidian.md/changelog/

Obsidian recently released the new feature Properties in versions v1.4.0 and v1.4.1 (insider builds), providing a new way to manage YAML front matters and supports several field types natively. In light of these changes, it may be necessary for the Projects plugin to take action in order to adapt to these new data types and ensure compatibility.

API changelog: https://github.com/obsidianmd/obsidian-api/commit/5178e27f0eb12741c79914f52146de8ff827e390

**Comment by marcusolsson** [comment]
Very excited about integrating Properties in the Projects flow! Hopefully it shouldn't need to much to accommodate.

**Reply by adamayers** [reply]
Hey there! I am keen to be able to generate projects based on properties like you can with Tags. For example I use a category property and I would like to be able to pull together a list based on the same category. Is that something you have on your radar?
